azure-proto-core/ArmClient.cs

```diff
@@ -22,15 +22,23 @@ public class ArmClient
     {
         static ArmClient()
         {
-            Registry.Register<ResourceGroupOperations, PhResourceGroup>(
-                new azure_proto_core.Internal.ArmResourceRegistration<ResourceGroupOperations, PhResourceGroup>(
+            Registry.Register(
+                new azure_proto_core.Internal.ArmResourceRegistration<ResourceGroupContainerOperations, SubscriptionOperations, ResourceGroupOperations, PhResourceGroup>(
                     new ResourceType("Microsoft.Resources/resourceGroups"),
                     (o, r) => new ResourceGroupContainerOperations(o, r),
                     (o, r) => new ResourceGroupOperations(o, r.Id)));
+
+            Registry.Register(
```

Comment:
I know I keep asking this, but what do we continue to use this for?

Reply:
This will be in a future change, just making the edits needed to get this task working.